update 20.treatments.md (#310)
greenelab/covid19-review: content/20.treatments.md

https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->

> This uncertainty is important to recognize clinically because HCQ-based treatments can lead to dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279]. Also, there is now an evidence of increased deaths on the use of HCQ in older men. Furthermore, administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients. [@doi:10.1101/2020.04.16.20065920]. 

Reviewer: ```suggestion
This uncertainty is important to recognize clinically because HCQ-based treatments can lead to dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279].
There is now evidence of increased mortality among older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients [@doi:10.1101/2020.04.16.20065920]. 
```
